docs/scripts_md/remove_jiv_data_from_db_and_filesystem.pl

```diff
@@ -0,0 +1,22 @@
+# NAME
+
+MakeArchiveLocationRelative.pl -- Removes the root directory from the
```

Comment:
mmmm, this is the wrong command to run for this script :)

Reply:
ooops. looks like I forgot to update the documentation... Fixed now